Updated `README.md`install instruction

dev1virtuoso opened the pull request:
Updated `README.md` install instruction to be more detailed

dev1virtuoso committed:
Create install.sh

dev1virtuoso committed:
Update README.md

dev1virtuoso committed:
Update README.md

dev1virtuoso committed:
Update install.sh

dev1virtuoso committed:
Update and rename install.sh to setup-eliza-ctss.sh

dev1virtuoso committed:
Update README.md

dev1virtuoso committed:
Create shutdown-ctss.sh

dev1virtuoso committed:
Create shutdown-ctss.sh

dev1virtuoso committed:
Delete shutdown-ctss.sh

dev1virtuoso committed:
Merge pull request #1 from dev1virtuoso/development

Development

dev1virtuoso committed:
Update setup-eliza-ctss.sh

dev1virtuoso committed:
Update setup-eliza-ctss.sh

dev1virtuoso committed:
Update README.md

dev1virtuoso committed:
Update setup-eliza-ctss.sh

dev1virtuoso committed:
Update setup-eliza-ctss.sh

dev1virtuoso committed:
Update README.md

dev1virtuoso committed:
Update setup-eliza-ctss.sh

dev1virtuoso committed:
Update setup-eliza-ctss.sh

dev1virtuoso committed:
Update setup-eliza-ctss.sh

dev1virtuoso committed:
Update setup-eliza-ctss.sh

dev1virtuoso committed:
Update setup-eliza-ctss.sh

dev1virtuoso committed:
Update setup-eliza-ctss.sh

dev1virtuoso committed:
Update setup-eliza-ctss.sh

dev1virtuoso committed:
Update setup-eliza-ctss.sh

dev1virtuoso committed:
Update setup-eliza-ctss.sh

dev1virtuoso committed:
Update README.md

dev1virtuoso committed:
Delete setup-eliza-ctss.sh

dev1virtuoso committed:
Delete shutdown-ctss.sh

dev1virtuoso committed:
Update README.md

rupertl commented:
Thanks for the PR! Having clear documentation is important.

The net effects of your changes are to points 2 and 5 in `README.md`. 

For point 2, I want to keep the details of how many times to press Enter etc in the output of `format-disks` and `install-disk-loader` rather than in the README to keep this short, so please revert this change.

For point 5, this change makes sense as a reminder for users to do this in a separate window. Could you reword this slightly to change 'Terminal' to 'terminal' as the capital-T form is only used on Mac and I want to keep this cross-platform.

dev1virtuoso committed:
Update README.md

dev1virtuoso commented:
Fixed Issues

1. Regarding point 2, reverted to the original version to fix a lengthy issue.
2. Regarding point 5, changed the word "Terminal" to "terminal" to maintain cross-platform compatibility.

Thanks for your suggestion.
